docs/howto/features/buckets.md

```diff
@@ -106,6 +106,35 @@ user_buckets = {
 }
 ```
 
+## Enabling Requester Pays flag on buckets
+
+Some hubs want to expose a particular bucket to the broad internet
+but not be billed for the charges associated with making and executing the
+requests on this bucket.
+
+By enabling the [Requester Pays flag](https://cloud.google.com/storage/docs/using-requester-pays#using),
+the requesters are required to include a billing project in their requests,
+which will mean that the billing will happen on the requester's project.
+
+Enabling Requester Pays is useful, for example, if the communities have a lot
+of data that they want to make available to others, but don't want to be
+charged for their access to that data.
+
+This can be enabled by setting the `requester_pays` parameter in `user_buckets`
+for the appropriate bucket, and running `terraform apply`.
+
+Example:
+
+```terraform
+user_buckets = {
+   "persistent": {
+      "delete_after": null,
+      "public_access": true,
+      "requester_pays": true
```

Comment:
@yuvipanda, do you think `requester_pays` has sense outside of a public access? Should these two flags be documented togheter?

Reply:
Ah, great question @GeorgianaElena.

Right now, we don't actually have a terraform variable called `requstor_pays` (see list for [gcp](https://github.com/2i2c-org/infrastructure/blob/083419739d40e4cb5fbd0e8ebdae9a59465b46ce/terraform/gcp/variables.tf#L255) and [aws](https://github.com/2i2c-org/infrastructure/blob/083419739d40e4cb5fbd0e8ebdae9a59465b46ce/terraform/aws/variables.tf#L25) for what we do have). So we currently *don't* have a way to mark a bucket as requestor pays via our terraform. 

It's not super hard to implement (pass through to [gcp config](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket#requester_pays) or [this aws resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_request_payment_configuration)), so we should implement it if / when a community asks to make one of their buckets into requestor_pays.

It may also be helpful to rename `public_access` to `public_anonymous_access` - basically, it currently means that *anyone* can access that bucket, without needing to provide any authentication at all. But since `requestor_pays` *requires* that the entity requesting this be part of an AWS or GCP project, it by definition is *authenticated* - so GCP or AWS knows who to charge! This 'authentication' is what our hubs provide when we turn on the `allow_access_to_requestor_pays_buckets` flag.

So `public_access` (or `public_anonymous_access`) is actually *mutually exclusive* with `requestor_pays`. The first is fully unauthenticated, while the second requires authentication.

I hope this is helpful!

Reply:
Extremely helpful @yuvipanda! Thank you very much for taking the time to clarify this <3